src/main/java/stsjorbsmod/relics/MindGlassRelic.java

```diff
@@ -50,10 +67,12 @@ public void onGainClarity(String id) {
                     new ApplyPowerAction(
                             AbstractDungeon.player,
                             AbstractDungeon.player,
-                            new MindGlassPower(AbstractDungeon.player, TEN_CLARITY_DAMAGE),
+                            new MindGlassPower(AbstractDungeon.player, TEN_CLARITY_DAMAGE, statTracker),
                             1,
                             true));
         }
+        PreAoeDamageStatsAction preAoeDamageStatsAction = new PreAoeDamageStatsAction();
+        AbstractDungeon.actionManager.addToBottom(preAoeDamageStatsAction);
         AbstractDungeon.actionManager.addToBottom(
                 new DamageAllEnemiesAction(
```

Comment:
If this damage kills all the enemies and ends the fight, the relic stats don't get updated properly. I think this might just need one of the calls below to be made in a different order or in a different spot.

Reply:
I thought I checked this scenario. It _should_ be.... Guess I'll take a look at this again.

Reply:
Well, there's a bug with the console command `clarify all` that doesn't trigger the `MindGlassPower`. Makes for testing this harder than expected.

So far, I haven't seen the 3 damage to all update incorrectly. This should basically be the same as what the RelicMod does. Could you show a scenario where you got that?

I'll keep looking into the 100 damage

Reply:
100 damage doesn't work quite right with Arcane Form and `memory clarify all`. filed #549

Reply:
well, there's an issue where it's not adding the "times 100 clarities correctly"

Reply:
It's a pain, but use the dev console to do:
```
memory clarify stsjorbsmod:.....
memory clarify stsjorbsmod:.....
memory clarify stsjorbsmod:.....
```